### ISSUE TITLE
how to store file content in a variable

### ISSUE THREAD

**Issue by exskube** [cyberark/conjur-cli#295]
Dear all,
I have searched around but was not able to find any reference to this topic. Can you please let me know how I can store a file content into a variable using the conjur cli? 
In the specific case, I need to store a p12 / keystore file, and using 
conjur variable values add path/to/variable $(cat path/to/p12) 
did not work well.
Thanks in advance for your help!
Giancarlo

**Comment by sgnn7** [comment]
Hey Giancarlo,
Storing files is not really supported as Conjur is intended to be a fast key-value store but if you really wanted to, you could in theory use [`base64`](https://linux.die.net/man/1/base64) to Base-64 encode it when storing and then `base64 -d` to decode after fetching. 

I think something like this should work though I didn't test it:

When storing:
```sh-session
conjur variable values add path/to/variable $(cat path/to/p12 | base64)
```

When retrieving:
```sh-session
conjur variable value path/to/variable | base64 -d > /target/path
```

I'll close this issue but feel free to reopen it if there is a legitimate bug in the CLI program.

Thanks,
Srdjan

**Comment by exskube** [comment]
Hey Srdjan,
thanks a lot for your reply! Actually thanks to your suggestion I managed to find a working solution adopting this workaround:
https://discuss.cyberarkcommons.org/t/can-i-store-a-file-as-a-secret/134/10
In addition to that, unfortunately I had to run summon on Windows 👎 to fetch the variable, and base64 decode and gunzip there are not working well, so I had to use git bash in order to obtain the desired result.
Thanks again!
Giancarlo
